docs/application/dotnet/guides/multimedia/media-playback.md

```diff
@@ -250,14 +250,14 @@ To play a video file:
 
 2. To set the display on which the video is played, use the `Display` property of the `Tizen.Multimedia.Player` class.
 
-    For example, to set the display on a Xamarin-based application, first create an instance of the [Tizen.Xamarin.Forms.Extension.MediaView](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Xamarin.Forms.Extension.MediaView.html) class, cast it to an instance of the [Tizen.Multimedia.MediaView](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.MediaView.html) class, and finally set that instance as the `Display` property:
+    For example, to set the display on a Xamarin-based application, first create an instance of the custom renderer(ex. VideoView) based on VisualElementRenderer class, cast it to an instance of the [Tizen.Multimedia.MediaView](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.MediaView.html) class, and finally set that instance as the `Display` property:
 
     ```
-    var mediaView = new Tizen.Xamarin.Forms.Extension.MediaView();
+    var mediaView = new VideoView();
 
     mediaView.NativeViewCreated += (s, e) =>
     {
-        player.Display = new Display((Tizen.Multimedia.MediaView)mediaView.NativeView);
+        player.Display = new Display((Tizen.Multimedia.MediaView)(s as VideoView).NativeView);
```

Comment:
Is it really "s as VideoView"?

Reply:
Yes. you can check this Tizen.Multimedia.Manual.Tests, support/CameraTestBase.cs